README.md

```diff
@@ -106,28 +92,16 @@ You should use one of these three ways to import the AWS Mobile SDK but not mult
     
         $ carthage update
 
-4. With your project open in Xcode, select your **Target**. Under **General** tab, find **Embedded Binaries** and then click the **+** button.
+4. With your project open in Xcode, select your **Target**. Under **General** tab, find **Frameworks, Libraries, and Embedded Content** and then click the **+** button.
 
-5. Click the **Add Other...** button, navigate to the `AWS<#ServiceName#>.framework` files under `Carthage` > `Build` > `iOS` and select them. Do not check the **Destination: Copy items if needed** checkbox when prompted.
+5. Click the **Add Other...** button, then "Add Files..." in the popup menu, then navigate to the `AWS<#ServiceName#>.framework` files under `Carthage` > `Build` > `iOS` and select them. Do not check the **Destination: Copy items if needed** checkbox if prompted.  Add the frameworks that you need for you specific use case.  For example, if you are using AWSMobileClient and AWSPinpoint, you will want to add the following frameworks:
 
-    * `AWSCore.framework`
-    * `AWSAutoScaling.framework`
-    * `AWSCloudWatch.framework`
-    * `AWSCognito.framework`
+    * `AWSAuthCore.framework`
```

Comment:
how do i know i need this just from needing AWSMobileClient?

Reply:
Spoke offline, resolving.